Package.swift

```diff
@@ -87,6 +87,26 @@ if ProcessInfo.processInfo.environment["SWIFTCI_INSTALL_RPATH_OS"] == "android"
  */
 let autoProducts = [swiftPMProduct, swiftPMDataModelProduct]
 
+private func useSwiftBuildPackageDependency() -> Bool {
+    return (ProcessInfo.processInfo.environment["SWIFTPM_SWBUILD_FRAMEWORK"] == nil &&
+    ProcessInfo.processInfo.environment["SWIFTPM_NO_SWBUILD_DEPENDENCY"] == nil)
+}
```

Comment:
Is there a particular reason this uses a function instead of a simple `let` binding? I also don't think `private` is needed here, there's no chance this function could be used accidentally elsewhere, since multi-file package manifest is not a thing.

Reply:
This is used in two places, so instead of duplicating the logic, I opted to put it in a function call.